webpack.config.js

```diff
@@ -98,9 +98,9 @@ if (process.env.NODE_ENV === 'production') {
     }),
     new CopyWebpackPlugin([
       { from: 'manifest.json', to: 'manifest.json' },
-      { from: 'coin.png', to: 'coin.png'},
-      { from: 'coins.html', to: 'coins.html' },
-      { from: 'icons/*.png', to: ''},
+      { from: 'assets/images/coin.png', to: 'coin.png'},
```

Comment:
@usmanity are you using these assets anywhere?

Reply:
yes, these get used as the extension's icons